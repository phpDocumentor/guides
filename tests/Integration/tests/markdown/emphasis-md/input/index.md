# Markdown with emphasis

*Italic* or _Italic_,
**Bold** or __Bold__,
**Bold and _Italic_**,
_Italic and **Bold**_

[_test_](https://phpdoc.org)
