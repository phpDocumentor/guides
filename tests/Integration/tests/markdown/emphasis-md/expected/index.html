<!-- content start -->
    <h1>Markdown with emphasis</h1>

<p><em>Italic</em> or <em>Italic</em><strong>Bold</strong> or <strong>Bold</strong></p>
<!-- content end -->
